crates/noirc_evaluator/src/ssa_refactor/ssa_builder/function_builder.rs

```diff
@@ -57,4 +60,49 @@ impl<'ssa> FunctionBuilder<'ssa> {
         let entry = self.current_function.entry_block();
         self.current_function.dfg.add_block_parameter(entry, typ)
     }
+
+    /// Insert a numeric constant into the current function
+    pub(crate) fn numeric_constant(&mut self, value: FieldElement, typ: Type) -> ValueId {
+        self.current_function.dfg.constant(value, typ)
+    }
+
+    /// Insert a numeric constant into the current function
```

Comment:
Should this say "with type Field"?

Reply:
Ah, yes. Copied the function and forgot to update the comment